src/client/styles/utils/colors.scss

```diff
@@ -0,0 +1,31 @@
+// colors
```

Comment:
In [PR 132](https://github.com/NYPL-discovery/discovery-front-end/pull/132), I put the .scss files from design toolkit into it's own `.src/client/styles/toolkit` folder.  I don't think it's a big deal since those files are temporary, but just flagging because it'll cause conflicts w/ the other branch once it's merged